addons/mail/static/src/activity/activity_list_popover_item.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.ActivityListPopoverItem" owl="1">
+        <div class="o-activity-list-popover-item d-flex flex-column border-bottom py-2">
+            <div class="o-activity-list-popover-item-container d-flex align-items-baseline ms-3 me-1">
+                <i t-if="props.activity.icon" class="fa small me-2" t-attf-class="{{ props.activity.icon }}" role="img"/>
+                <t t-if="props.activity.summary">
+                    <b class="text-900 me-2 text-truncate flex-grow-1 flex-basis-0" t-esc="props.activity.summary"/>
+                </t>
+                <t t-if="!props.activity.summary and props.activity.activity_type_id">
+                    <b class="text-900 me-2 text-truncate flex-grow-1" t-esc="props.activity.activity_type_id[1]"/>
+                </t>
+                <button t-if="hasEditButton" class="o-activity-list-popover-item-edit-button btn btn-sm btn-link" t-on-click="onClickEditActivityButton">
+                    <i class="fa fa-pencil"/>
+                </button>
+                <t t-if="props.activity.can_write">
+                    <button t-if="hasFileUploader" class="o-activity-list-popover-item-action-link btn btn-link shadow-none fs-4 fa fa-upload" title="Upload file" aria-label="Upload File" t-on-click="onClickUploadDocument"/>
+                    <button t-if="hasMarkDoneButton" class="o-activity-list-popover-item-action-link o_ActivityListViewItem_markAsDone btn btn-link shadow-none fs-4 fa fa-check-circle" title="Mark as done" aria-label="Mark as done" t-on-click="onClickMarkAsDone" t-ref="markDoneButton"/>
+                </t>
+            </div>
+            <div t-if="props.activity.state !== 'today'" class="d-flex align-items-baseline flex-wrap mx-3">
+                <i class="fa fa-clock-o me-2 text-muted" role="img" aria-label="Deadline" title="Deadline"/>
+                <t t-if="!props.activity.isCurrentPartnerAssignee and props.activity.assignee">
```

Comment:
I fail to see where `isCurrentPartnerAssignee` could be present on `activity`. Dead code?

Reply:
adapted